packages/core/src/interfaces/event-listener.interface.ts

```diff
@@ -20,9 +12,8 @@ export interface EventListenerInterface {
     supports<T>(event: Event<T>): boolean;
 
     /**
-     * This method receives an event and handles it and returns an EventResponse.
+     * This method receives an event and handles it.
      * @param event
-     * @param eventResponse The event response that will in the end be returned.
      */
-    handle<EventPayload, EventResponsePayload>(event: Event<EventPayload>, eventResponse: EventResponse<EventPayload, EventResponsePayload>): Promise<EventResponse<EventPayload, EventResponsePayload>>;
+    handle<EventPayload>(event: Event<EventPayload>): Promise<void>;
```

Comment:
Should we change handle for something else since we now have handlers ?

Reply:
Yes good point. Maybe execute